site/blog/2022/nodejs-deploy-codepipeline.mdx

```diff
@@ -0,0 +1,222 @@
+---
+slug: deploy-nodejs-app-with-codepipeline
+title: Deploy a NodeJS App on AWS using CodePipeline
```

Comment:
and CodeDeploy?

Reply:
for this blogpost i took the approach of using CodePipeline, as it was automating the source and the deploy action. CodeDeploy is used, but wrapped into CodePipeline. Do you think that is worth mentioning that in the title?